### PR TITLE
feat: add a sample default trasformer

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: Mehant Kammakomati <kmehant@gmail.com>

### CODE REVIEW

**Comment by kmehant** [comment]
@HarikrishnanBalagopal Please let me know if this is enough for being a default transformer sample. Thank you.

**Comment by kmehant** [comment]
Screenshot 2022-09-28 at 8 12 28 PM

**Comment by kmehant** [comment]
Screenshot 2022-09-28 at 8 16 43 PM

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: custom-default-transformer/defaulttransformer.star [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Example for default transformer
+def transform(new_artifacts, old_artifacts):
+    pathMappings = []
+    artifacts = []
+    data = {}
+    transformerName = "Default Transformers"
```
[the remark this answers is not in the document]
Comment (kmehant) [reply custom-default-transformer/defaulttransformer.star]: Fixed

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**